docs/Data_Portal/Users_Guide/Developers_Guide.md

```diff
@@ -430,28 +474,29 @@ Finally, the following hooks are available for querying set size:
 * `useSsmSetCountsQuery`
 * `useCaseSetCountsQuery`
 
-## Creating a cohort
+## Creating a Cohort
 
-Depending on what your application does, you may want to create a new cohort. Although the GDC Portal SDK provides a
-number of functions for creating a new cohort. It is highly recommended that the application use the provided Button and
-SaveCohortModal components to create a new cohort. The Button and SaveCohortModal components are located in
+Depending on the application function, if may be beneficial to create a new cohort. Although the GDC Data Portal SDK provides a
+number of functions for creating a new cohort. It is highly recommended that the application use the provided `Button` and
+`SaveCohortModal` components to create a new cohort. The `Button` and `SaveCohortModal` components are located in
```

Comment:
```suggestion
Depending on the application function, it may be beneficial to create a new cohort. Although the GDC Data Portal SDK provides a
number of functions for creating a new cohort, it is highly recommended that the application use the provided `Button` and
`SaveCohortModal` components to create a new cohort. The `Button` and `SaveCohortModal` components are located in
```